source/extensions/common/aws/credential_providers/iam_roles_anywhere_credentials_provider.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+#include "envoy/extensions/common/aws/v3/credential_provider.pb.h"
+
+#include "source/extensions/common/aws/aws_cluster_manager.h"
+#include "source/extensions/common/aws/metadata_credentials_provider_base.h"
+#include "source/extensions/common/aws/metadata_fetcher.h"
+#include "source/extensions/common/aws/signers/iam_roles_anywhere_sigv4_signer_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Aws {
+
+// IAM Roles Anywhere credential strings
+constexpr char CREDENTIAL_SET[] = "credentialSet";
+constexpr char CREDENTIALS_LOWER[] = "credentials";
+constexpr char ACCESS_KEY_ID_LOWER[] = "accessKeyId";
+constexpr char SECRET_ACCESS_KEY_LOWER[] = "secretAccessKey";
+constexpr char EXPIRATION_LOWER[] = "expiration";
+constexpr char SESSION_TOKEN_LOWER[] = "sessionToken";
+
+constexpr char ROLESANYWHERE_SERVICE[] = "rolesanywhere";
```

Comment:
Is this being used in the source part of the code?
If only in tests, consider moving it there.